src/custom/components/ErrorBoundary/ErrorBoundaryMod.tsx

```diff
@@ -69,32 +135,34 @@ export default class ErrorBoundary extends React.Component<unknown, ErrorBoundar
     if (error !== null) {
       const encodedBody = encodeURIComponent(issueBody(error))
       return (
-        <FallbackWrapper>
-          <BodyWrapper>
+        <AppWrapper>
+          <HeaderWrapper>
+            <a href=".">
+              <CowLogo>
+                <LogoImage />
+              </CowLogo>
+            </a>
+          </HeaderWrapper>
+          <Wrapper>
+            <FlexContainer>
+              <StyledTitle>
+                <Trans> Something went wrong</Trans>
+              </StyledTitle>
+              <img src={CowError} alt="CowSwap Error" height="125" />
+            </FlexContainer>
             <AutoColumn gap={'md'}>
-              <SomethingWentWrongWrapper>
-                <TYPE.label fontSize={24} fontWeight={600}>
-                  <Trans>
-                    <span role="img" aria-label="cow-exclamation">
-                      🐮❕
-                    </span>{' '}
-                    Something went wrong
-                  </Trans>
-                </TYPE.label>
-              </SomethingWentWrongWrapper>
               <CodeBlockWrapper>
                 <code>
                   <TYPE.main fontSize={10} color={'text1'}>
-                    {error.stack}
+                    <StyledParagraph>{error.stack}</StyledParagraph>
                   </TYPE.main>
```

Comment:
After a review along with @matextrem, we came to the conclusion that it is not a CSS problem.
It is related how the error is interpreted and rendered in iOS (Safari and Chrome). The complete message is not stored directly in the error, as it happens in other platforms and devices. 

For example on Safari we get:
```
GI@https://pr1259--gpswapui.review.gnosisdev.com/static/js/main.1856fc78.chunk.js:1:585424 ro@https://pr1259--gpswapui.review.gnosisdev.com/static/js/4.262e7757.chunk.js:2:2473681 Js@https://pr1259--gpswapui.review.gnosisdev.com/static/js/4.262e7757.chunk.js:2:2526140 Su@https://pr1259--gpswapui.review.gnosisdev.com/static/js/4.262e7757.chunk.js:2:2513314 Eu@https://pr1259--gpswapui.review.gnosisdev.com/static/js/4.262e7757.chunk.js:2:2513242 Tu@https://pr1259--gpswapui.review.gnosisdev.com/static/js/4.262e7757.chunk.js:2:2513105 mu@https://pr1259--gpswapui.review.gnosisdev.com/static/js/4.262e7757.chunk.js:2:2510071 mu@[native code] https://pr1259--gpswapui.review.gnosisdev.com/static/js/4.262e7757.chunk.js:2:2459449 https://pr1259--gpswapui.review.gnosisdev.com/static/js/4.262e7757.chunk.js:2:2536868 Gi@https://pr1259--gpswapui.review.gnosisdev.com/static/js/4.262e7757.chunk.js:2:2459395 Ji@https://pr1259--gpswapui.review.gnosisdev.com/static/js/4.262e7757.chunk.js:2:2459330 fu@https://pr1259--gpswapui.review.gnosisdev.com/static/js/4.262e7757.chunk.js:2:2507440 Oo@https://pr1259--gpswapui.review.gnosisdev.com/static/js/4.262e7757.chunk.js:2:2479007 Oo@[native code] https://pr1259--gpswapui.review.gnosisdev.com/static/js/4.262e7757.chunk.js:2:281441 promiseReactionJob@[native code]
```
![image](https://user-images.githubusercontent.com/622217/130681219-1420edfd-9383-454d-a2dd-453aba008206.png)




Instead of:

```
Error: This cow is dropping some serious milk 🥛!!
    at CowGamePage (http://localhost:3000/static/js/main.chunk.js:43233:11)
    at renderWithHooks (http://localhost:3000/static/js/vendors~main.chunk.js:236075:22)
    at mountIndeterminateComponent (http://localhost:3000/static/js/vendors~main.chunk.js:238837:17)
    at beginWork (http://localhost:3000/static/js/vendors~main.chunk.js:240036:20)
    at HTMLUnknownElement.callCallback (http://localhost:3000/static/js/vendors~main.chunk.js:225025:18)
    at Object.invokeGuardedCallbackDev (http://localhost:3000/static/js/vendors~main.chunk.js:225074:20)
    at invokeGuardedCallback (http://localhost:3000/static/js/vendors~main.chunk.js:225134:35)
    at beginWork$1 (http://localhost:3000/static/js/vendors~main.chunk.js:244876:11)
    at performUnitOfWork (http://localhost:3000/static/js/vendors~main.chunk.js:243712:16)
    at workLoopSync (http://localhost:3000/static/js/vendors~main.chunk.js:243649:9)
    at renderRootSync (http://localhost:3000/static/js/vendors~main.chunk.js:243615:11)
    at performSyncWorkOnRoot (http://localhost:3000/static/js/vendors~main.chunk.js:243232:22)
    at http://localhost:3000/static/js/vendors~main.chunk.js:232485:30
    at unstable_runWithPriority (http://localhost:3000/static/js/vendors~main.chunk.js:307490:16)
    at runWithPriority$1 (http://localhost:3000/static/js/vendors~main.chunk.js:232431:14)
    at flushSyncCallbackQueueImpl (http://localhost:3000/static/js/vendors~main.chunk.js:232480:13)
    at flushSyncCallbackQueue (http://localhost:3000/static/js/vendors~main.chunk.js:232468:7)
    at discreteUpdates$1 (http://localhost:3000/static/js/vendors~main.chunk.js:243360:13)
    at discreteUpdates (http://localhost:3000/static/js/vendors~main.chunk.js:224835:16)
    at dispatchDiscreteEvent (http://localhost:3000/static/js/vendors~main.chunk.js:227017:7)
```